webapp/src/components/propertyValueElement.tsx

```diff
@@ -38,11 +39,15 @@ const PropertyValueElement = (props:Props): JSX.Element => {
             return !isNaN(parseInt(val, 10))
         case 'email': {
             const emailRegexp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-            return emailRegexp.test(val.toLowerCase())
+            return emailRegexp.test(lowerCaseVal)
```

Comment:
there is no need of making to lowercase for email or url, the regular expresions are allowing uppercase/lowercase characters anyway.

Reply:
Makes sense. I'll make changes. Thanks for the review!

Reply:
also removed `lowercase` for email and url @ https://github.com/mattermost/focalboard/pull/286/commits/d87b90e6b932e2cf7693871d26e9bad6a3efda42

Reply:
also removed `lowercase` for email and url @ https://github.com/mattermost/focalboard/pull/286/commits/d87b90e6b932e2cf7693871d26e9bad6a3efda42